### PR TITLE
Revamps the Zipkin Architecture overview

### PR DESCRIPTION
**Pull request by adriancole**
This rewrites the instrumentation process overview per experience giving
presentations and insights captured in #23

This is intentionally completing one section vs rewriting the whole web
site. That's to help us progress incrementally.

See also https://speakerdeck.com/adriancole/introduction-to-distributed-tracing-and-zipkin-at-devopsdays-singapore

### CODE REVIEW

**Comment by adriancole** [comment]
cc @openzipkin/core @mjbryant @shakuzen @nicmunroe

File: pages/architecture.md [hunk 1]
```diff
@@ -3,22 +3,36 @@ title: Architecture
...
 
+Tracers live in your applications and record timing and metadata about
+operations that that took place. They often instrument libraries, so that their
+use is transparent to users. For example, an instrumented web server records
+when it received a request and when it sent a response. The trace data collected
+is called a Span.
 
-Instrumented libraries
-----------------------
+Instrumentation are written to be safe in production and have little overhead.
```
Comment (mjbryant) [inline pages/architecture.md]: Nit: either "Instrumentation is" or "Instrumentations are". In the former case you'd have to also do s/have/has.

File: pages/architecture.md [hunk 2]
```diff
@@ -3,22 +3,36 @@ title: Architecture
...
+The component in an instrumented app that sends data to Zipkin is called a
+Reporter. Reporters send trace data via one of several transports to Zipkin
+collectors, which persist trace data to storage. Later, storage is queried by
+the API to provide data to the UI.
 
-Tracing information is collected on each host using the instrumented libraries
-and sent to Zipkin. When the host makes a request to another service, it passes
-a few tracing identifiers along with the request so we can later tie the data
-together.
+Here's an diagram describing this flow:
```
Comment (mjbryant) [inline pages/architecture.md]: Nit: s/an/a

**Comment by adriancole** [comment]
@mjbryant thanks for the review

**Pull request merged**